BigQuery cohort execution FAILED NoClassDefFoundError: com/google/auth/http/HttpTransportFactory

WebApi 2.8.1
### Expected behavior
Analysis finished successfully

### Actual behavior
We get the following error:
`2021-02-24 13:58:03.343 ERROR taskExecutor-6 org.springframework.batch.core.step.AbstractStep -  - Encountered an error executing step cohortDefinition.generateCohort in job generateCohort
java.lang.NoClassDefFoundError: com/google/auth/http/HttpTransportFactory
	at com.simba.googlebigquery.googlebigquery.core.BQConnection.connect(Unknown Source)
	at com.simba.googlebigquery.jdbc.common.BaseConnectionFactory.doConnect(Unknown Source)
	at com.simba.googlebigquery.jdbc.common.AbstractDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:208)
	at org.springframework.jdbc.datasource.DriverManagerDataSource.getConnectionFromDriverManager(DriverManagerDataSource.java:153)
	at org.springframework.jdbc.datasource.DriverManagerDataSource.getConnectionFromDriver(DriverManagerDataSource.java:144)
	at org.springframework.jdbc.datasource.AbstractDriverBasedDataSource.getConnectionFromDriver(AbstractDriverBasedDataSource.java:196)
	at org.springframework.jdbc.datasource.AbstractDriverBasedDataSource.getConnection(AbstractDriverBasedDataSource.java:159)
	at org.ohdsi.webapi.util.TempTableCleanupManager.lambda$cleanupTempTables$0(TempTableCleanupManager.java:61)
	at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:133)
	at org.ohdsi.webapi.util.TempTableCleanupManager.cleanupTempTables(TempTableCleanupManager.java:59)
	at org.ohdsi.webapi.service.GenerationTaskExceptionHandler.handleException(GenerationTaskExceptionHandler.java:17)
	at org.springframework.batch.repeat.support.RepeatTemplate.doHandle(RepeatTemplate.java:300)
	at org.springframework.batch.repeat.support.RepeatTemplate.executeInternal(RepeatTemplate.java:220)
	at org.springframework.batch.repeat.support.RepeatTemplate.iterate(RepeatTemplate.java:144)
	at org.springframework.batch.core.step.tasklet.TaskletStep.doExecute(TaskletStep.java:257)
	at org.springframework.batch.core.step.AbstractStep.execute(AbstractStep.java:200)
	at org.springframework.batch.core.job.SimpleStepHandler.handleStep(SimpleStepHandler.java:148)
	at org.springframework.batch.core.job.AbstractJob.handleStep(AbstractJob.java:392)
	at org.springframework.batch.core.job.SimpleJob.doExecute(SimpleJob.java:135)
	at org.springframework.batch.core.job.AbstractJob.execute(AbstractJob.java:306)
	at org.springframework.batch.core.launch.support.SimpleJobLauncher$1.run(SimpleJobLauncher.java:135)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: com.google.auth.http.HttpTransportFactory`

### Steps to reproduce behavior
1)Run any cohort on BigQuery data source

--- a/pom.xml
+++ b/pom.xml
@@ -1345,6 +1345,16 @@
           <artifactId>google-cloud-bigquerystorage</artifactId>
           <version>0.101.0-beta</version>
         </dependency>
+        <dependency>
+          <groupId>com.google.auth</groupId>
+          <artifactId>google-auth-library-oauth2-http</artifactId>
+          <version>0.16.2</version>
+        </dependency>
+        <dependency>
+          <groupId>com.google.auth</groupId>
+          <artifactId>google-auth-library-credentials</artifactId>
+          <version>0.16.2</version>
+        </dependency>
       </dependencies>
       <build>
         <plugins>

